### DISCUSSION TITLE
Continue Series Section

### DISCUSSION THREAD

**Discussion by endangered-maritime-walrus** [advplyr/audiobookshelf#538] [Q&A]
Thanks for all of your hard work. It has quickly become one of my favorite apps.

What happened to the continue series section? I have not seen it for a while now, or am I missing it?

**Comment by advplyr** [comment]
Everything that used series had to be re-written in v2 when we allowed books to belong to multiple series.
This home page category just hasn't been rewritten and until you no one brought it up so I wasn't sure if it was still a useful shelf.

If I remember correctly that shelf would find series that are in progress and show the next book in the series that has not been started yet. Is that right?

**Reply by advplyr** [reply]
Added in [v2.0.8](https://github.com/advplyr/audiobookshelf/releases/tag/v2.0.8)

**Reply by Cobolt78** [reply]
> Everything that used series had to be re-written in v2 when we allowed books to belong to multiple series. This home page category just hasn't been rewritten and until you no one brought it up so I wasn't sure if it was still a useful shelf.
> 
New user here and I can say I love the continue series :)

**Comment by endangered-maritime-walrus** [comment]
I assumed it had to do with the multiple series addition. I didn't notice how much I used it til it was missing. I seem to jump from one series to another and seeing a list of ones that I am in the middle of was really great.

> If I remember correctly that shelf would find series that are in progress and show the next book in the series that has not been started yet. Is that right?

That is correct. Related to #331
